### PR TITLE
Removed query string from URL for metrics

### PR DESCRIPTION
**Pull request by gmsdelmundo**

### CODE REVIEW

File: src/main/java/com/uid2/operator/Main.java [hunk 1]
```diff
@@ -353,21 +353,21 @@ private static void setupMetrics(MicrometerMetricsOptions metricOptions) {
...
-                .meterFilter(MeterFilter.deny(id ->
-                    id.getName().startsWith(MetricsDomain.HTTP_SERVER.getPrefix()) &&
-                    Objects.equals(id.getTag(Label.HTTP_CODE.toString()), "404")))
-                // adding common labels
-                .commonTags("application", "uid2-operator");
+                    // providing common renaming for prometheus metric, e.g. "hello.world" to "hello_world"
+                    .meterFilter(new PrometheusRenameFilter())
+                    .meterFilter(MeterFilter.replaceTagValues(Label.HTTP_PATH.toString(), actualPath -> {
+                        try {
+                            return HttpUtils.normalizePath(actualPath).split("\\?")[0];
```
Comment (scong-ttd) [inline src/main/java/com/uid2/operator/Main.java:360]: why "\\?" instead of "?"

**Review by scong-ttd — COMMENTED** [review]
NIT: since we don't force a code formatting let's follow the existing formatting and avoid whitespace changes :)

File: src/main/java/com/uid2/operator/Main.java (hunk 1 above)
Comment (gmsdelmundo) [reply src/main/java/com/uid2/operator/Main.java:360]: `.split()` takes in regex, so ? needs to be escaped

**Comment by gmsdelmundo** [comment]
> NIT: since we don't force a code formatting let's follow the existing formatting and avoid whitespace changes :)

Updated

File: src/main/java/com/uid2/operator/Main.java (hunk 1 above)
Comment (scong-ttd) [reply src/main/java/com/uid2/operator/Main.java]: 👍

**Review by scong-ttd — APPROVED** [review]

**Pull request merged**